WoloxAndroidBootstrp/app/src/main/java/ar/com/wolox/android/example/network/services/PostService.kt

```diff
@@ -1,12 +1,23 @@
 package ar.com.wolox.android.example.network.services
 
+import ar.com.wolox.android.example.model.LoginResponse
 import ar.com.wolox.android.example.model.Post
 import retrofit2.Response
+import retrofit2.http.Field
 import retrofit2.http.GET
+import retrofit2.http.POST
 import retrofit2.http.Path
+import retrofit2.http.FormUrlEncoded
 
 interface PostService {
 
     @GET("/posts/{id}")
     suspend fun getPostById(@Path("id") id: Int): Response<Post>
+
+    @FormUrlEncoded
+    @POST("/auth/sign_in")
+    suspend fun loginUser(
+        @Field("email") email: String,
+        @Field("password") password: String
+    ): Response<LoginResponse>
```

Comment:
You should move it to its own file, according to the feature. For instance, LoginService.
Also, you don't need to specify every field manually. Just try passing a data class that has email and password as fields, and using the @Body annotation instead of @Field. Something like:
`suspend fun loginUser(@Body loginRequest: LoginRequest): Response<LoginResponse>`